### ISSUE TITLE
Metal Posts from Railcraft are missing the recipes for recoloring

### ISSUE THREAD

**Issue by HDR** [xJon/Tekkit-2#68]
**Basic details**
- **Are you using a legitimate Minecraft account?**: yes
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**: not applicable

The recipes needed to recolor Railcraft metal posts are missing, you can craft the black metal posts, but they can not be recolored, the intended recipe seems to be to surround a piece of dye with metal posts.

CraftTwaker recipes that fix this issue (using vanilla dyes) if anyone wants a quick fix for themselves:

```
// Add recipes for Railcraft colored metal posts
recipes.addShaped("White Metal Post", <railcraft:post_metal:0> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:15>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);
                                        
recipes.addShaped("Orange Metal Post", <railcraft:post_metal:1> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:14>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Magenta Metal Post", <railcraft:post_metal:2> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:13>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Light Blue Metal Post", <railcraft:post_metal:3> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:12>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Yellow Metal Post", <railcraft:post_metal:4> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:11>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Lime Metal Post", <railcraft:post_metal:5> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:10>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Pink Metal Post", <railcraft:post_metal:6> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:9>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Gray Metal Post", <railcraft:post_metal:7> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:8>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Light Gray Metal Post", <railcraft:post_metal:8> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:7>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Cyan Metal Post", <railcraft:post_metal:9> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:6>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Purple Metal Post", <railcraft:post_metal:10> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:5>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Blue Metal Post", <railcraft:post_metal:11> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:4>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Brown Metal Post", <railcraft:post_metal:12> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:3>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Green Metal Post", <railcraft:post_metal:13> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:2>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Red Metal Post", <railcraft:post_metal:14> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye:1>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);

recipes.addShaped("Black Metal Post", <railcraft:post_metal:15> * 8,
										[[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <minecraft:dye>, <railcraft:post_metal:*>],
										[<railcraft:post_metal:*>, <railcraft:post_metal:*>, <railcraft:post_metal:*>]]);
```

**Comment by xJon** [comment]
This is caused by https://github.com/Railcraft/Railcraft/issues/2042, and will get fixed once Railcraft has a new release thanks to https://github.com/Railcraft/Railcraft/pull/2049. Thanks nonetheless for reporting this!

**Comment by xJon** [comment]
This is resolved in version `1.1.0` :)
